apps/staking/src/pages/blocks/[[...block]].tsx

```diff
@@ -123,25 +124,39 @@ const Blocks: FC<BlocksProps> = ({ chainId }) => {
     blockId = blockId && blockId.length > 0 ? (blockId[0] as string) : '';
 
     const [searchKey, setSearchKey] = useState<string>(blockId);
+    const [debouncedSearchKey, setDebouncedSearchKey] = useState(searchKey);
```

Comment:
Hey @nevendyulgerov, 
I think we could eliminate the `searchKey` and start the `debouncedSearchKey` with the `blockId` value, also replacing the use of `searchKey` and `setSearchKey` below. It looks redundant.